commands/volumes/common.go

```diff
@@ -23,22 +23,33 @@ func (e invalidOptionError) Error() string {
 	return e.option
 }
 
+func splitOptionName(option string) (string, string, string) {
```

Comment:
It will be nice to have this function exported, and possibly in a more common place, as I'd like to use it in volgen as well. We would need a volume-set or an options package to perform option validation. This function will be a nice addition in it.

Reply:
I'll move this to `utils` package or `volume/volume-utils.go` for now. The `xlator` package currently has code such as definition of `Option` struct. There's no clear conceptual distinction between an xlator option and a volume option. Practically, most of the times they mean the same. We can consolidate these into one package in future.